src/Service/JsonResponseBuilderException.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the OpenapiBundle package.
+ *
+ * (c) Niels Nijens <nijens.niels@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nijens\OpenapiBundle\Service;
+
+use Exception;
+use Nijens\OpenapiBundle\Exception\HttpExceptionInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+/**
+ * Builds a JSON response to an exception for OpenAPI routes.
+ *
+ * @author David Cochrum <dcochrum@sonnysdirect.com>
+ */
+class JsonResponseBuilderException implements JsonResponseBuilderExceptionInterface
```

Comment:
Why did you decide to change the name from the previous `ExceptionJsonResponseBuilder` to this? At the moment the name would suggest this class is actually an exception class.

Reply:
I was going back and forth on the naming ... in the master branch of our fork, I ended up with `ExceptionJsonResponseBuilder` whereas this branch was somehow left with `JsonResponseBuilderException` ... does `ExceptionJsonResponseBuilder` or `JsonResponseExceptionBuilder` (seems to more closely match `JsonResponseExceptionSubscriber`) read better to you?

Reply:
I'd still go with `ExceptionJsonResponseBuilder` as it's a `JsonResponseBuilder` for exceptions. I guess `JsonResponseExceptionSubscriber` could also use a better name, but that's for another time. :-)